client/src/main/java/org/spine3/base/Identifiers.java

```diff
@@ -61,10 +61,11 @@ private Identifiers() {
      * The type of the value wrapped into the returned instance is defined by the type
      * of the passed value:
      * <ul>
-     *      <li>For classes implementing {@link com.google.protobuf.Message} — the value of the message itself
-     *      <li>For {@code String} — {@link com.google.protobuf.StringValue}
-     *      <li>For {@code Long} — {@link com.google.protobuf.UInt64Value}
-     *      <li>For {@code Integer} — {@link com.google.protobuf.UInt32Value}
+     *      <li>For classes implementing {@link com.google.protobuf.Message Message} — the value
+     *      of the message itself
```

Comment:
The same as I mentioned before.
Could you move `the value` to the next line to decrease the differences between the line lengths.